src/deepsparse/transformers/eval_downstream.py

```diff
@@ -604,7 +675,24 @@ def parse_args():
         type=bool,
         default=False,
     )
-
+    parser.add_argument(
```

Comment:
this is only supported for text gen as of this PR - should add a comment and a check or warning to make the user aware

Reply:
Fixed